pypeit/wavetilts.py

```diff
@@ -294,108 +330,153 @@ def run(self, maskslits=None, doqa=True, debug=False, show=False):
+                                                               self.slitcen[:,slit], slit,
+                                                               debug=debug)
             thismask = self.slitmask == slit
             # Trace
-            self.trace_dict = self.trace_tilts(self.msarc, self.lines_spec, self.lines_spat, thismask, self.slitcen[:,slit])
+            self.trace_dict = self.trace_tilts(self.msarc, self.lines_spec, self.lines_spat,
+                                               thismask, self.slitcen[:,slit])
             #if show:
             #    ginga.show_tilts(viewer, ch, self.trace_dict)
 
             self.spat_order[slit] = self._parse_param(self.par, 'spat_order', slit)
             self.spec_order[slit] = self._parse_param(self.par, 'spec_order', slit)
             # 2D model of the tilts, includes construction of QA
-            coeff_out = self.fit_tilts(self.trace_dict, thismask, self.slitcen[:,slit], self.spat_order[slit],
-                                       self.spec_order[slit], slit,doqa=doqa, show_QA = show, debug=show)
+            coeff_out = self.fit_tilts(self.trace_dict, thismask, self.slitcen[:,slit],
+                                       self.spat_order[slit], self.spec_order[slit], slit,
+                                       doqa=doqa, show_QA=show, debug=show)
             self.coeffs[0:self.spec_order[slit]+1, 0:self.spat_order[slit]+1 , slit] = coeff_out
-            # Tilts are created with the size of the original slitmask, which corresonds to the same binning
-            # as the science images, trace images, and pixelflats etc.
-            self.tilts = tracewave.fit2tilts(self.slitmask_science.shape, coeff_out, self.par['func2d'])
+
+            # Tilts are created with the size of the original slitmask,
+            # which corresonds to the same binning as the science
+            # images, trace images, and pixelflats etc.
+            self.tilts = tracewave.fit2tilts(self.slitmask_science.shape, coeff_out,
+                                             self.par['func2d'])
             # Save to final image
             thismask_science = self.slitmask_science == slit
             self.final_tilts[thismask_science] = self.tilts[thismask_science]
 
-        self.tilts_dict = {'tilts':self.final_tilts, 'coeffs':self.coeffs, 'slitcen': self.slitcen, 'func2d':self.par['func2d'],
-                           'nslit': self.nslits, 'spat_order': self.spat_order, 'spec_order': self.spec_order}
+        self.tilts_dict = {'tilts':self.final_tilts, 'coeffs':self.coeffs, 'slitcen':self.slitcen,
+                           'func2d':self.par['func2d'], 'nslit':self.nslits,
+                           'spat_order':self.spat_order, 'spec_order':self.spec_order}
         return self.tilts_dict, maskslits
 
-    def load_master(self, filename, exten=0, force=False):
+    def save(self, outfile=None, overwrite=True):
```

Comment:
optional arguments required if I want to save something not created by the class, i.e. tilts_dict = None or something.

Reply:
Only done for `WaveImage` and `TraceSlits` as a temporary stop-gap.  Not implemented generally.

Reply:
Okay that is fine.

Reply:
Okay, good.  Thanks.